tests/ert_tests/ert3/config/test_stages_config.py

```diff
@@ -250,12 +255,13 @@ def test_stage_input_unknown_field(base_stage_config):
 
 def test_stage_output_immutable(base_stage_config):
     config = ert3.config.load_stages_config(base_stage_config)
-
+    output_record_name = base_stage_config[0]["output"][0]["record"]
     with pytest.raises(TypeError, match="does not support item assignment"):
-        config[0].output[0] = None
+        config[0].output[output_record_name] = None
 
     with pytest.raises(TypeError, match="does not support item assignment"):
-        config[0].output[0].record = None
+        print(type(output_record_name), type(config[0].output))
```

Comment:
Debugging?

Reply:
Actually unsure how this sneaked in, but removed in future commit.